modin/pandas/__init__.py

```diff
@@ -90,9 +90,8 @@
 
 # Set this so that Pandas doesn't try to multithread by itself
 os.environ["OMP_NUM_THREADS"] = "1"
-DEFAULT_NPARTITIONS = 4
-num_cpus = 1
 
+num_cpus = 1
```

Comment:
This should be removed because no used anywhere. Also, please remove it [here](https://github.com/modin-project/modin/blob/master/modin/pandas/test/test_api.py#L65).

Reply:
There were many uses of it, removed all of them.